pkg/scheduler/worker/workerpool_test.go

```diff
@@ -87,6 +88,63 @@ func TestWorkerPool(t *testing.T) {
 	requireOpsProcessableExists(t, opsProcessable, testInvoker.params[0].CorrelationID)
 }
 
+func TestWorkerPoolMaxOpRetriesReached(t *testing.T) {
+	test.IntegrationTest(t) //required because a valid Kubeconfig is required to create test cluster entry
+
+	//create cluster inventory
+	inventory, err := cluster.NewInventory(db.NewTestConnection(t), true, &cluster.MetricsCollectorMock{})
+	require.NoError(t, err)
+
+	//add cluster to inventory
+	clusterState, err := inventory.CreateOrUpdate(1, kebTest.NewCluster(t, "1", 1, false, kebTest.OneComponentDummy))
```

Comment:
let's choose a new `runtimeID`, not use `"1"` which is used in another test in case some data is not cleaned up.

Reply:
That's a good idea! It's done!

Reply:
done